rotkehlchen/chain/zksync_lite/manager.py

```diff
@@ -61,6 +62,7 @@ def __init__(
             database: 'DBHandler',
     ) -> None:
         self.database = database
+        self.msg_aggregator = database.msg_aggregator
```

Comment:
no need to create a new variable for this. Access self.database.msg_aggregator directly